README.md

```diff
@@ -68,9 +67,18 @@ If you are planning on doing mod development using UE4SS, you can do the same as
 
 There are several different ways you can build UE4SS.
 
-### Building from cli
+## Building from cli
```

Comment:
This isn't for building from CLI

Reply:
It seems to me that it is.
It goes through how you would use the command line interface of xmake to build UE4SS, and it then also goes through how you generate a VS solution.

Could you explain what you mean exactly ?
Is it that the `Opening in an IDE` section is inside the `Building from cli` section instead of its own section that's the problem ?

Reply:
It only sets the configuration, it doesn't mention building from CLI at all.

Reply:
Yes it does, right here: https://github.com/UE4SS-RE/RE-UE4SS/pull/516/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R79